pkgs/beef-farm/mock/beef_farm_mock.go

```diff
@@ -5,8 +5,9 @@
 package mock_beeffarm
 
 import (
```

Comment:
why did you regenerate mocks? how it's related to this PR?

Reply:
because they didn't work for me :(

Reply:
Hm, ok